titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/DefaultKubeApiFacade.java

```diff
@@ -119,8 +148,9 @@ private void activate() {
                         apiClient
                 );
 
-                this.nodeInformer = createNodeInformer(sharedInformerFactory, coreV1Api);
-                this.podInformer = createPodInformer(sharedInformerFactory, coreV1Api);
+                this.nodeInformer = createNodeInformer(sharedInformerFactory);
```

Comment:
Since these are now centrally managed, would be nice to have gauges that show their current size.

Reply:
I am working on metrics in another PR. I will include this as well.